controllers/neutronapi_controller.go

```diff
@@ -1683,6 +1804,20 @@ func (r *NeutronAPIReconciler) generateServiceSecrets(
 	templateParameters["MemcachedTLS"] = mc.GetMemcachedTLSSupport()
 	templateParameters["TimeOut"] = instance.Spec.APITimeout
 
+	if instance.Status.NotificationsTransportURLSecret != nil {
+		notificationsTransportURL, _, err := secret.GetDataFromSecret(
```

Comment:
maybe we could update `getTransportURL()` to be also used for the notifications url. like passing in instance.Status.NotificationsTransportURLSecret as a parameter